src/6model/reprs/MVMStaticFrameSpesh.c

```diff
@@ -95,25 +77,25 @@ static MVMuint64 unmanaged_size(MVMThreadContext *tc, MVMSTable *st, void *data)
     for (spesh_idx = 0; spesh_idx < body->num_spesh_candidates; spesh_idx++) {
         MVMSpeshCandidate *cand = body->spesh_candidates[spesh_idx];
 
-        size += cand->bytecode_size;
+        size += cand->body.bytecode_size;
 
-        size += sizeof(MVMFrameHandler) * cand->num_handlers;
+        size += sizeof(MVMFrameHandler) * cand->body.num_handlers;
 
-        size += sizeof(MVMCollectable *) * cand->num_spesh_slots;
+        size += sizeof(MVMCollectable *) * cand->body.num_spesh_slots;
 
-        size += sizeof(MVMint32) * cand->num_deopts;
+        size += sizeof(MVMint32) * cand->body.num_deopts;
 
-        size += sizeof(MVMSpeshInline) * cand->num_inlines;
+        size += sizeof(MVMSpeshInline) * cand->body.num_inlines;
 
-        size += sizeof(MVMuint16) * (cand->num_locals + cand->num_lexicals);
+        size += sizeof(MVMuint16) * (cand->body.num_locals + cand->body.num_lexicals);
 
         /* XXX probably don't need to measure the bytecode size here,
          * as it's probably just a pointer to the same bytecode we have in
          * the static frame anyway. */
 
         /* Dive into the jit code */
-        if (cand->jitcode) {
-            MVMJitCode *code = cand->jitcode;
+        if (cand->body.jitcode) {
+            MVMJitCode *code = cand->body.jitcode;
```

Comment:
This lot is wrong now, I think; the spesh candidate is its own GC-able object, and the unmanaged memory size is accounted for there. So we should not account for it in this object also.